README_LD.md

```diff
@@ -0,0 +1,74 @@
+Takeaway Challenge
+==================
+```
+                            _________
+              r==           |       |
+           _  //            |  M.A. |   ))))
+          |_)//(''''':      |       |
+            //  \_____:_____.-------D     )))))
+           //   | ===  |   /        \
+       .:'//.   \ \=|   \ /  .:'':./    )))))
+      :' // ':   \ \ ''..'--:'-.. ':
+      '. '' .'    \:.....:--'.-'' .'
+       ':..:'                ':..:'
+
+ ```
+
+Summary
+-----
```

Comment:
Your read me is extremely detailed and clear, which helps a lot with understanding what you have done and why. I like that you have summarised what each class does and the methods, variables etc it contains.